<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumManager</class>
 <widget class="QDialog" name="AlbumManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuseScore: Album Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Album Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="albumName">
       <property name="toolTip">
        <string>album name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="remove">
       <property name="toolTip">
        <string>remove current score</string>
       </property>
       <property name="text">
        <string>Remove Score</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="4">
      <widget class="QListWidget" name="scoreList">
       <property name="toolTip">
        <string>score list</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="down">
       <property name="toolTip">
        <string>move current score down in list</string>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="up">
       <property name="toolTip">
        <string>move current score up in list</string>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scoreName">
       <property name="toolTip">
        <string>path to new score you want to add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fileDialog">
       <property name="toolTip">
        <string>open file manager</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="createNew">
       <property name="toolTip">
        <string>create new album</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load">
       <property name="toolTip">
        <string>load an existing album</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="print">
       <property name="text">
        <string>Print Album</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createScore">
       <property name="text">
        <string>Join Scores</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlbumManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlbumManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
